src/Microsoft.Health.Dicom.Functions/Update/UpdateDurableFunction.Orchestration.cs

```diff
@@ -110,6 +110,7 @@ public partial class UpdateDurableFunction
                     Errors = input.Errors,
                     CreatedTime = input.CreatedTime ?? await context.GetCreatedTimeAsync(_options.RetryOptions),
                 });
+            _updateMeter.UpdatedInstances.Add(instanceWatermarks.Count);
```

Comment:
this should be above line: 102

Reply:
this should be above line: 102. Also could you add a unit test that we have called the meter

Reply:
Done